lib/philomena/tag_changes/query.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Philomena.TagChanges.Query do
+  alias PhilomenaQuery.Parse.Parser
+
+  defp user_my_transform(%{user: %{id: id}}, "changes"),
+    do: {:ok, %{term: %{true_user_id: id}}}
+
+  defp user_my_transform(_ctx, _value),
+    do: {:error, "Unknown `my' value."}
+
+  defp anonymous_fields do
+    [
+      int_fields: ~W(id tags_count added_tags_count removed_tags_count),
+      numeric_fields: ~W(user_id image_id tag_ids added_tag_ids removed_tag_ids),
+      date_fields: ~W(created_at),
+      literal_fields: ~W(user tags added_tags removed_tags),
+      bool_fields: ~W(anonymous),
+      default_field: {"tags", :term},
+      aliases: %{
+        "added" => "added_tags",
```

Comment:
why do we need aliases in a new index?

Reply:
I assumed this is how I make both "added_tags" and "added" fields do the same thing. I wasn't sure if just naming the index "added" to begin with isn't too confusing.